common/action-manager.js

```diff
@@ -146,10 +145,6 @@ function NotifyUpdateSearchString(searchString) {
   return Notify("NOTIFY_UPDATE_SEARCH_STRING", {searchString}, {skipMasterStore: true});
 }
 
-function RequestExperiments() {
```

Comment:
These is just some unused stuff, since we don't actually request experiments any more (they just get dispatched directly to the store)